packages/server/src/http/getHTTPStatusCode.ts

```diff
@@ -1,12 +1,12 @@
-import { TRPCError } from '../../error/TRPCError';
-import { invert } from '../../internals/invert';
-import { TRPCResponse, TRPC_ERROR_CODES_BY_KEY } from '../../rpc';
+import { TRPCError } from '../error/TRPCError';
+import { invert } from '../internals/invert';
+import { TRPCResponse, TRPC_ERROR_CODES_BY_KEY } from '../rpc';
 
 export const TRPC_ERROR_CODES_BY_NUMBER = invert(TRPC_ERROR_CODES_BY_KEY);
 type ValueOf<TType> = TType[keyof TType];
 
 export type TRPC_ERROR_CODE_NUMBER = ValueOf<typeof TRPC_ERROR_CODES_BY_KEY>;
-const JSONRPC2_TO_HTTP_CODE: Record<
+export const JSONRPC2_TO_HTTP_CODE: Record<
```

Comment:
I'm still keen on getting rid of these arbitrary numbers that are in our RPC spec just because we're following the JSON-RPC which I don't know how useful it even is

```suggestion 
const JSONRPC2_TO_HTTP_CODE: Record<
```